soh/soh/OTRGlobals.cpp

```diff
@@ -1223,6 +1303,8 @@ extern "C" void Graph_StartFrame() {
     }
 #endif
     OTRGlobals::Instance->context->GetWindow()->StartFrame();
+
+    processDroppedFiles();
```

Comment:
does this need to run always on every frame? i know with rando we only run the logic on file select which prevents any possible performance issues we might encounter by running it during gameplay

Reply:
Until we have a more event based approach, I'm not sure there's a better place for it, I'd rather not have to tell people to go to a specific screen for it to load their configuration

Reply:
Moved to every frame on the file select screen for the time being. Will circle back later on once we have a more event based approach.